File: PHP/Product.php
<?php

class Product
{
    private $idProduct = "Lorem 3000";
    private $price = 1000;

    //Construct
    public function __construct($idProduct = "Lorem 3000", $price = 1000){
        $this->idProduct = $idProduct;
        $this->price = $price;
    }

    /*Get-Set*/
    public function setIdProduct($idProduct){
        $this->idProduct = $idProduct;
    }

    public function getIdProduct(){
        return $this->idProduct;
    }

    public function setPrice($price){
        $this->price = $price;
    }

    public function getPrice(){
        return $this->price;
    }
}

//Tampil Data Produk
function tampilDataPR($Product){
    echo "Data Produk" ."<br>";
    echo "ID: " .$Product->getIdProduct() ."<br>";
    echo "Harga: " .$Product->getPrice() ."<br> ======== <br>";
}

?>

File: CPP/product.cpp
#include <bits/stdc++.h>
#include <iostream>
#include <string>
using namespace std;

class Product{

    private:
        int price;
        string idProduct;

    public:
        Product(){
        }

        void setPrice(int price){
            this->price = price;
        }
        int getPrice(){
            return this->price;
        }

        void setIdProduct(string idProduct){
            this->idProduct = idProduct;
        }
        string getIdProduct(){
            return this->idProduct;
        }

        ~Product(){
        }
};
